packages/canary-web-components/src/components/ic-tree-item/ic-tree-item.tsx

```diff
@@ -0,0 +1,385 @@
+import {
+  Component,
+  h,
+  Prop,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Watch,
+  State,
+  Listen,
+  forceUpdate,
+  Method,
+} from "@stencil/core";
+import { IcSizes, IcThemeForegroundNoDefault } from "../../utils/types";
+import {
+  isSlotUsed,
+  onComponentRequiredPropUndefined,
+  checkSlotInChildMutations,
+} from "../../utils/helpers";
+import arrowDropdown from "../../assets/arrow-dropdown.svg";
+
+let treeItemIds = 0;
+
+/**
+ * @slot label - Content is placed as the tree item label.
```

Comment:
"Placed" doesn't feel like quite the right word here but might just be me - maybe:
```suggestion
 * @slot label - Content is set as the tree item label.
```